packages/daisy/src/components/card/card.tsx

```diff
@@ -0,0 +1,18 @@
+import { component$, HTMLAttributes, Slot } from '@builder.io/qwik';
+import {
+  Card as HeadlessCard,
+} from '@qwik-ui/headless';
+import { clsq } from '@qwik-ui/shared';
+
+export type CardProps = HTMLAttributes<HTMLElement>;
+
+export const Card = component$(
+  (props: CardProps) => {
+    const { class: cls, ...rest } = props;
```

Comment:
```suggestion
    const { class: cssClasses, ...rest } = props;
```

Reply:
Ok thank you for the advice.
I renamed it to `className` to looks like the DOM property name and use the same name already used by another component: `headless\..\tabs.tsx`